src/org/zaproxy/zap/extension/history/HistoryFilterPlusDialog.java

```diff
@@ -486,8 +479,8 @@ private JScrollPane getUrlRegxExcScroller() {
 	private JScrollPane getTagScroller() {
 		if (tagScroller == null) {
 			tagList = new JList<>(getTagModel());
+			tagList.setPrototypeCellValue("Tags are short...");

```

Comment:
Just a placeholder to let the list component calculate a (more) reasonable width.